packages/schema/index.ts

```diff
@@ -140,3 +140,46 @@ export const AddressSchema = z.object({
 // const identifier = "PoolMetadataSchema";
 // export const { node } = zodToTs(PoolMetadataSchema, identifier);
 // export const typeAlias = createTypeAlias(node, identifier);
+
+export const getStableSwapSimulatorTokensSchema = ({
+  symbolToEdit,
+  existentSymbols,
+}: {
+  symbolToEdit?: string;
+  existentSymbols?: string[];
+}) => {
+  const StableSwapSimulatorTokensSchema = z.object({
+    symbol: z
```

Comment:
wouldn't `z.enum(existentSymbols)` be enough? Also I think you'll lose the advantage that Zod would give you in generating the TS type if you pass `existentSymbols` as a string array. Not sure I fully get why this needs to be a standalone function -- are you planning on having multiple lists be attributable to this schema?